ownership_test.go

```diff
@@ -24,3 +25,17 @@ func TestOwnershipInvalidUser(t *testing.T) {
 	ownership := NewOwnership("invalid", "invalid")
 	assert.NotNil(t, ownership, "should never return nil")
 }
+
+func TestGroupFileParsingValid(t *testing.T) {
+	file, err := ioutil.TempFile("", "keywhiz-fs-test")
+	panicOnError(err)
+
+	_, err = file.WriteString("test:x:1234:test0,test1\n")
```

Comment:
I'd prefer if we had 3-4 lines in here and checked we could read all of them, but it's no big deal.